release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -55,6 +55,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-16-rhcos"]
 === {op-system-first}
 
+==== {op-system} now uses {op-system-base} 9.4
+
+{op-system} now uses {op-system-base-full} 9.4 packages in {product-title} {product-version}. These packages ensure that your {product-title} instances receives the latest fixes, features, enhancements, hardware support, and driver updates. Excluded from this change, {product-title} 4.14 is an Extended Update Support (EUS) release that will continue to use {op-system-base} 9.2 EUS packages for the entirety of its lifecycle.
```

Comment:
```suggestion
{op-system} now uses {op-system-base-full} 9.4 packages in {product-title} {product-version}. These packages ensure that your {product-title} instances receive the latest fixes, features, enhancements, hardware support, and driver updates. As an Extended Update Support (EUS) release,{product-title} 4.14 is excluded from this change and will continue to use {op-system-base} 9.2 EUS packages for the entirety of its lifecycle.
```
- Fix typo
- Consider the edit I made to the second sentence. Not sure if I changed the meaning, but as it's currently worded it seems a bit awkward. Lmk if you have questions.

Reply:
this does sound better, thanks Brian!